### PR TITLE
Fix travis

### LINKED ISSUE #149: Fix Travis tests
**I'm submitting a ...**
- [X] bug report

**Current behavior:**

Currently, travis is failing.
https://travis-ci.org/fossasia/badgeyay/builds/278248341?utm_source=github_status&utm_medium=notification

**Expected behavior:**

Travis should run all the tests #147 and succeed.

**Steps to reproduce:**

Please create a pull request to work on this issue and submit commits which fix the errors.

### PR DESCRIPTION
**Pull request by mohdomama**
Fixes #149 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Make Travis tests run with firefox headless
- Make flask run in background with : ` python app/main.py >> log.txt 2>&1 & `
- Here is the link of the travis built for my local fork :
 https://travis-ci.org/mohdomama/badgeyay/builds/290547521

### CODE REVIEW

**Comment by S2606** [comment]
@mohdomama Please squash your commits.

File: app/tests/test.py [hunk 1]
```diff
@@ -4,14 +4,14 @@
...
 
 class BadgeyayTest(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
         if is_travis:
-            cls.driver = webdriver.PhantomJS()
+            cls.driver = webdriver.Firefox()
         else:
             cls.driver = webdriver.Firefox()
```
Comment (S2606) [inline app/tests/test.py]: Remove the if-else condition as there is no need

Comment (mohdomama) [reply app/tests/test.py]: Yeah sure working on it

**Comment by S2606** [comment]
Great work @mohdomama ..

**Comment by mohdomama** [comment]
@S2606 Thank you for your review. :smile:

**Comment by vaibhavsingh97** [comment]
@mohdomama awesome work!!

**Pull request merged**